applications/project_aurras_mvp_phase_2.md

```diff
@@ -0,0 +1,239 @@
+
+**Shreyas Kura Subramanya**  
+Shreyas is an experienced software developer with great zeal in implementing solutions in the software domain. He is skilled in Blockchain, smart contracts, ethical hacking, Rust and Golang. He currently is the Development Team Lead at HugoByte AI Labs and is instrumental in driving the team to great success. He obtained his Bachelor's degree focused in Computer Science from Bapuji Institute of Engineering & Technology, Davanagere. 
+ 
+**Shanith K K**  
+Shanith is a Software Engineer and a Web developer. Having graduated his M.Sc in Software Engineering, he is a diligent and industrious technocrat. He has published a paper titled on "Segmentation: A review" in JETIR (JETIRCG06006). He has worked on machine learning using python programming. He has substantial and working knowledge about programming languages like Rust, Python, C, C++, and he is currently working on blockchain development in HugoByte AI Labs. He has an insatiable passion for coding and is exceptionally good at problem-solving.
+
+### Team Code Repos
+
+- https://github.com/hugobyte
+- https://github.com/HugoByte?q=aurras
+- https://github.com/MuhammedIrfan
+- https://github.com/HanumantappaBudihal
+- https://github.com/shreyasbhat0
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/muhammed-irfan-k
+- https://www.linkedin.com/in/hanumantappa-budihal/
+- https://www.linkedin.com/in/shreyas-ks/
+- https://www.linkedin.com/in/shanith-k-k-09378988/
+- https://www.linkedin.com/in/amogha-sudarshan/
+- https://www.linkedin.com/in/immanuel-stanley-436456218/
+
+## Development Status :open_book:
+
+#### Article
+- https://medium.com/hugobyte/completion-of-aurras-mvp-a-middleware-for-the-web3-ecosystem-ee4f666be6aa
+
+#### Documentation
+- https://docs.aurras.hugobyte.com/
+
+#### Codebase
+- https://github.com/HugoByte/aurras
+- https://github.com/HugoByte/aurras-deployment-docker-compose
+- https://github.com/HugoByte/aurras-event-feed-substrate-js
+- https://github.com/HugoByte/aurras-deployment-kubernetes
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Description** Development of Project Aurras - MVP - Phase 2 
+- **Total Estimated Duration:** 268 Person Days
+- **Full-time equivalent (FTE):** 4
+- **Total Costs:** 42880 USD 
+
+### Milestone 1 — Workflow Composer - Part 1
```

Comment:
Are you going to deliver docker files that allow for running your deliverables? If so, it would be nice if you could add them to the milestone deliveries. See also our [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md#development-roadmap-nut_and_bolt):

> Deliverables 0a-0d are mandatory for all milestones

Reply:
Sure takahser we will update the deliverables.

Reply:
@MuhammedIrfan are you delivering docker files only for this delivery or also M1 and M3?

Reply:
@takahser we will have it only for M2 as its the only executable left without docker file, Rest of the docker files has been delivered as part of phase 1.